war/src/main/js/pluginSetupWizardGui.js

```diff
@@ -136,11 +136,19 @@ var createPluginSetupWizard = function(appendTarget) {
 			success.apply(this, arguments);
 		};
 	};
+	
+	var pluginList;
+	var allPluginNames;
+	var selectedPluginNames;
+	
+	// Initialize the plugin manager after connectivity checks
+	pluginManager.init(handleGenericError(function() {
```

Comment:
This probably should happen just after loading translations, in a wrapped block like I did it. It has the potential not to load before the user clicks a button to proceed doing something with plugins.